uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences-manager-leave-requests.js

```diff
@@ -45,6 +45,40 @@ module.exports = (function () {
       });
 
       return this;
-    }
+    },
+    /**
+     * Opens leave type filter
+     * @param {Number} leaveType index like 2 for TOIL etc.,
```

Comment:
Please document all the possible values, like
0. Absence
1. Sickness
2. Overtime/TOIL